umh.docs.umh.app/content/en/docs/architecture/_index.md

```diff
@@ -280,8 +282,8 @@ mechanisms.
 
 {{< mermaid theme="neutral" >}}
```

Comment:
also you need to generate an SVG / PNG from it so your changes are also shown to the user

Reply:
See the "Additional information" section of the PR's description